packages/headless/src/integration-tests/field-suggestions.test.ts

```diff
@@ -0,0 +1,253 @@
+import {
```

Comment:
This file contains functional tests instead of unit tests. I thought it would be more useful since we don't have a field suggestions component in Atomic which we can do end to end tests on.

Although this file contains functional tests, they use the real API.